internal/export/distro/format.go

```diff
@@ -38,7 +37,8 @@ func (jsonTr jsonFormatter) Format(event *models.Event) []byte {
 
 	b, err := json.Marshal(event)
 	if err != nil {
-		logger.Error("Error parsing JSON", zap.Error(err))
+		LoggingClient.Error(err.Error())
+		LoggingClient.Debug("Error parsing JSON")
```

Comment:
Should be better to have one log message instead of two. There are more cases like this.